### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by username-forGithub**
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.
- Added partials containing the following information:
_user_post
_post
_user

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+  <% @user.recent_three.each do |post| %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "posts/post", post: post %>
+    <% end %>
+  <% end %>
+  
+  <%= link_to user_posts_path(@user), class: 'self-center' do %>
+    <button type="button" class="see-all-posts mt-4 button text-white bg-gradient-to-r from-blue-500 to-blue-200 hover:shadow-none shadow-none border-2 hover:border-blue-400 hover:text-blue-400 hover:from-white hover:to-white">See all posts</button>
+  <% end %>
+</section>
```
Comment (DammyShittu) [inline app/views/users/show.html.erb]: - [ ] Kindly improve your styling for the user show page. Kindly center your items on the page based on the wireframe.

  See [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABY4RVENW6H3RCPXJM4QS73IY4KPT6Q)

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @username-forGithub,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by username-forGithub** [comment]
Amended. Centered.

**Comment by arslanbisharat** [comment]
Hi @username-forGithub,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by username-forGithub** [comment]
Thank you.
@arslanbisharat  and @DammyShittu

**Pull request merged**